### ISSUE TITLE
Build error on DragonFlyBSD

### ISSUE THREAD

**Issue by tdfbsd** [Azure/azure-storage-blob-go#112]
% go get github.com/Azure/azure-storage-blob-go/azblob           
```
# github.com/Azure/azure-storage-blob-go/azblob
go/src/github.com/Azure/azure-storage-blob-go/azblob/highlevel.go:144:7: undefined: mmf
go/src/github.com/Azure/azure-storage-blob-go/azblob/highlevel.go:146:12: undefined: newMMF
go/src/github.com/Azure/azure-storage-blob-go/azblob/highlevel.go:272:13: undefined: newMMF

```
Adding "dragonfly" to the // +build line of azblob/zc_mmf_unix.go fixes it.

**Comment by zezha-msft** [comment]
Hi @tdfbsd, thanks for the feedback!

I'll add it for the next release.

**Comment by tdfbsd** [comment]
Thanks, do you have a schedule for the next release?

**Comment by zezha-msft** [comment]
Hi @tdfbsd it's fixed in `v0.6.0`. Thank you!
